exchanges/btcmarkets/testdata/wsAllTrades.json

```
@@ -0,0 +1,3 @@
+{"marketId":"BTC-AUD","timestamp":"2025-03-13T08:27:55.691Z","tradeId":7006384466,"price":"131200.34","volume":"0.00151228","messageType":"trade","side":"Bid"}
+{"marketId":"BTC-AUD","timestamp":"2025-03-13T08:28:02.273Z","tradeId":7006384467,"price":"131065.01","volume":"0.05","messageType":"trade","side":"Ask"}
+{"marketId":"BTC-AUD","timestamp":"2025-03-13T08:28:02.274Z","tradeId":7006384468,"price":"131065.02","volume":"0.06","messageType":"trade","side":"Invalid"}
```

🚧 Need a test for A valid but not acceptable side to hit the default clause
```suggestion
{"marketId":"BTC-AUD","timestamp":"2025-03-13T08:28:02.274Z","tradeId":7006384468,"price":"131065.02","volume":"0.06","messageType":"trade","side":"Pentagon"}
{"marketId":"BTC-AUD","timestamp":"2025-03-13T08:28:02.2745","tradeId":7006384469,"price":"131060.02","volume":"0.07","messageType":"trade","side":"Any"}
```